src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -32,6 +35,25 @@ public OverseasEntitySubmissionDtoValidator(EntityNameValidator entityNameValida
 
     public Errors validateFull(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
 
+        if (isRoeUpdateEnabled && StringUtils.isNotBlank(overseasEntitySubmissionDto.getEntityNumber())) {
+            validateUpdateDetails(overseasEntitySubmissionDto, errors, loggingContext);
+        } else {
+            validateRegistrationDetails(overseasEntitySubmissionDto, errors, loggingContext);
+        }
+        return errors;
+    }
+
+    private void validateUpdateDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
+        // Method to be added to as Update journey developed
+        validateCommonDetails(overseasEntitySubmissionDto, errors, loggingContext);
+    }
+
+    private void validateRegistrationDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
+        validateCommonDetails(overseasEntitySubmissionDto, errors, loggingContext);
+        ownersAndOfficersDataBlockValidator.validateOwnersAndOfficersAgainstStatement(overseasEntitySubmissionDto, errors, loggingContext);
+    }
+
+    private void validateCommonDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
```

Comment:
As the validateCommonDetails is called as part of validateFull, I'd include the word Full in teh method name as it won't be applicable for partialValidations
eg validateFullCommonDetails(...)